src/core/Updater.cpp

```diff
@@ -130,9 +133,12 @@ Transfer* Updater::downloadFile(const QUrl &url, const QString &path)
 {
 	const QString urlString(url.path());
 	Transfer *transfer(TransfersManager::startTransfer(url, path + urlString.mid(urlString.lastIndexOf(QLatin1Char('/')) + 1), (Transfer::CanOverwriteOption)));
-	transfer->setParent(this);
+	if (transfer)
```

Comment:
Same here, there should be always a blank line before and after `if...else` block etc., unless it is next to start or end of parent block.